ios/FluentUI/Navigation/SearchBar.swift

```diff
@@ -184,7 +184,7 @@ open class SearchBar: UIView {
     // user interaction point
     private lazy var searchTextField: SearchBarTextField = {
         let textField = SearchBarTextField()
-        textField.font = Fonts.body.withSize(Constants.fontSize)
+        textField.font = UIFont.fluent(fluentTheme.aliasTokens.typography[.body1]).withSize(Constants.fontSize)
```

Comment:
Nit: could just say `.fluent` instead of `UIFont.fluent` here and elsewhere (doesn't need to be changed now, but for future reference)